Src/IronPython.Modules/nt.cs

```diff
@@ -402,18 +402,22 @@ public static class PythonNT {
         }
 
 #if FEATURE_UNIX && FEATURE_NATIVE
```

Comment:
Hmm, how do we distribute the Unix build?

Reply:
We're not currently

Reply:
Travis doesn't have the capability to archive artifacts (at least last time I looked), so there isn't an easy way to distribute a release built on Travis. It would be super nice if there was a way.

Reply:
Since the use of FEATURE_UNIX is very limited I wonder if we could just change them all to runtime checks and have the same binaries for all platforms. Either that or make the unix version buildable on Windows so we can include it in the generated nuget.

Reply:
I don't want to make Mono.Posix a reference on Windows platforms. That's my only hangup with that idea.